lang/printer/src/types.rs

```diff
@@ -19,6 +19,7 @@ pub enum Anno {
     BraceOpen,
     BraceClose,
     Error,
+    Reference(String, String),
```

Comment:
I would suggest using a record variant as follows:

```suggestion
    Reference { module_uri: Url, name: String },
```

where `module_uri` is the URI of the module, not including the anchor `#name`, yet.